### PR TITLE
Change type for Base

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json [hunk 1]
```diff
@@ -1,16 +1,13 @@
 {
     "name": "Tether",
-    "website": "https://tether.to",
-    "description": "Tether (USDT) is a cryptocurrency with a value meant to mirror the value of the U.S. dollar.",
-    "explorer": "https://evm.confluxscan.net/address/0xfe97e85d13abd9c1c33384e796f10b73905637ce",
-    "type": "CFXEVM",
     "symbol": "USDT",
+    "type": "CFXEVM",
```
Comment (vcoolish) [inline blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json:4]:
```suggestion
    "type": "CONFLUX",
```

File: blockchains/cfxevm/assets/0xa47f43DE2f9623aCb395CA4905746496D2014d57/info.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
     "name": "Ethereum",
-    "website": "https://ethereum.org/",
+    "symbol": "ETH",
+    "type": "CFXEVM",
```
Comment (vcoolish) [inline blockchains/cfxevm/assets/0xa47f43DE2f9623aCb395CA4905746496D2014d57/info.json:4]:
```suggestion
    "type": "CONFLUX",
```

File: blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json [hunk 3]
```diff
@@ -1,17 +1,14 @@
 {
     "name": "USD Base Coin",
-    "website": "https://www.centre.io",
+    "symbol": "USDC",
+    "type": "BASE20",
```
Comment (vcoolish) [inline blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json:4]:
```suggestion
    "type": "BASE",
```

File: blockchains/base/assets/0x78a087d713Be963Bf307b18F2Ff8122EF9A63ae9/info.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "Baseswap",
-    "website": "https://baseswap.fi",
-    "description": "Decentralized finance leverages the individual and collective capacity of all of us, without interference.Join us on BASE chain now.",
-    "explorer": "https://basescan.org/token/0x78a087d713Be963Bf307b18F2Ff8122EF9A63ae9",
-    "type": "BASE20",
     "symbol": "BSWAP",
+    "type": "BASE20",
```
Comment (vcoolish) [inline blockchains/base/assets/0x78a087d713Be963Bf307b18F2Ff8122EF9A63ae9/info.json:4]:
```suggestion
    "type": "BASE",
```

File: blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json (hunk 1 above)
Comment (defisaur) [reply blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json:4]: @vcoolish in database we have CFXEVM for now. Should we rename it to CONFLUX ?

File: blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json (hunk 3 above)
Comment (defisaur) [reply blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json:4]: @vcoolish in database we have BASE20 for now. Should we rename it to BASE ?

Comment (cryptomanz1) [reply blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json:4]: @defisaur yes, please

Comment (cryptomanz1) [reply blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json:4]: @defisaur Yes, please

File: blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json (hunk 1 above)
Comment (cryptomanz1) [reply blockchains/cfxevm/assets/0xfe97E85d13ABD9c1c33384E796F10B73905637cE/info.json:4]: @defisaur Yes, please

**Comment by defisaur** [comment]
@cryptomanz1 I see there is no CI errors, so you can merge, I already prepared db rename query and we will execute it today

**Comment by defisaur** [comment]
> @cryptomanz1 I see there is no CI errors, so you can merge, I already prepared db rename query and we will execute it today

Done

**Pull request merged**